ext/tidy/tests/bug77040.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Bug #77040 (tidyNode::isHtml() is completely broken)
+--SKIPIF--
+<?php
+if (!extension_loaded('tidy')) die('skip tidy extension not available');
+?>
+--FILE--
+<?php
+$tidy = new tidy;
+$tidy->parseString("<p>text</p>");
+$p = $tidy->body()->child[0]->child[0];
+var_dump($p->type === TIDY_NODETYPE_TEXT);
+var_dump($p->isHtml());
```

Comment:
Can you please also add a negative test (where it returns false)?

Reply:
I have added a check for the `<p>` element (which succeeds), and a check for CDATA (which fails).

With the current code (and libtidy 5.6.0), only the CDATA check would fail.